haystack/retriever/dense.py

```diff
@@ -54,29 +69,52 @@ def __init__(self,
                             The title is expected to be present in doc.meta["name"] and can be supplied in the documents
                             before writing them to the DocumentStore like this:
                             {"text": "my text", "meta": {"name": "my title"}}.
-        :param remove_sep_tok_from_untitled_passages: If embed_title is ``True``, there are different strategies to deal with documents that don't have a title.
```

Comment:
Our default behaviour here is now remove_sep_tok_from_untitled_passages=True, right?
Maybe we should add a TODO in the code to do further experiments which setting is better here...

Reply:
We don't remove [SEP] tokens from untitled passages anymore. In TextSimilarity processor, untitled passages are encoded as: [CLS][SEP][SEP][1st token][2nd token]....[SEP]